test/validate/eia_test.py

```diff
@@ -40,94 +39,3 @@ def test_no_null_cols_eia(pudl_out_eia, live_dbs, cols, df_name):
     pv.no_null_cols(
         pudl_out_eia.__getattribute__(df_name)(), cols=cols, df_name=df_name
     )
-
-
-@pytest.mark.parametrize(
-    "df_name,unique_subset",
-    [
-        (
-            "bf_eia923",
-            Resource.from_id("core_eia923__monthly_boiler_fuel").schema.primary_key,
```

Comment:
These are all being generated from the PK constraints, and so can definitely be removed.